src/examples/InstrumentList.ts

```diff
@@ -0,0 +1,7 @@
+interface InstrumentList extends Context {
+  type: 'fdc3.instrumentList',
+  id: {
+    instrumentListId?: string;
```

Comment:
I have a question about the id here. Should we leave this out? I think the instrumentList should just be an array of instruments, it doesn't preclude people from adding an id. "InsturmentListId" is quite specific.

Reply:
Agree with Riko, to specific and we don't preclude from adding ID's.
Could in some scenarios make sense with a name as well